xml/slurm.xml

```diff
@@ -249,31 +249,31 @@ sbatch sleeper.sh
     dependence of the job which a user has run in the history.
    </para>
    <para>
-    The slurm database has two components which is the daemon
-    <literal>slurmdbd</literal> itself and a mysql database, where
-    <literal>mariab</literal> is recommended. The database can be on the same
+    The slurm database has two components: the <literal>slurmdbd</literal>
+    daemon itself, and a MySQL database, where <literal>mariab</literal> is
```

Comment:
1. If you use a MySQL database under MariaDB, I guess you could say it's a MariaDB database..?
2. The word "where" does not really seem to fit into this sentence.
3. `s/<literal>mariadb<\/literal>/MariaDB/`?